frontend/src/store/modules/maps.js

```diff
@@ -109,7 +109,6 @@ const actions = {
         apiFunc = metabolitesApi.fetchMetaboliteData;
         break;
       default:
-        // TODO: handle unexpected type
         break;
```

Comment:
What's the meaning to have a  `break` here when the default action is empty?
We can either delete
```
default:
  break;
```
or we assign the default value of apiFunc to e.g. `null`. 
But when `apiFunc` is undefined or null (which seems will never happen), the function `apiFunc` should not be called.

Reply:
You are right that a `break` is not needed here. Sometimes people leave it in the switch statements, such as [this example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#syntax).

The `apiFunc` variable must always be one of `gene`, `reaction`, or `metabolite` and it should always be called whenever the `getSelectedElement` function is called. There is some room for improvement here to make things clearer. Overall it felt like there are too many instances of this level of improvement in the project and this one in specific didn't warrant a dedicated issue.

Reply:
Good to get verified that `apiFunc` will always be assigned a value. I'm ok to leave the code as is.